model/riscv_sys_control.sail

```diff
@@ -557,10 +557,6 @@ function init_sys() -> unit = {
    * See riscv_vlen.sail for details.
    */
   vstart             = zero_extend(0b0);
-  vxsat              = 0b0;
-  vxrm               = 0b00;
-  vcsr[vxrm]       = vxrm;
-  vcsr[vxsat]      = vxsat;
```

Comment:
This might want to be `vcsr[vxrm] = 0b00` and `vcsr[vxsat] = 0b0` rather than removing both.